File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Coordinates.java
package com.riders.thelab.data.remote.dto.weather;

import com.squareup.moshi.Json;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Coordinates {

    @Json(name = "lon")
    double longitude;

    @Json(name = "lat")
    double latitude;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Coordinates.kt
package com.riders.thelab.data.remote.dto.weather

import kotlinx.serialization.SerialName
import java.io.Serializable

@kotlinx.serialization.Serializable
data class Coordinates(
    @SerialName("lon")
    val longitude: Double = 0.0,

    @SerialName("lat")
    val latitude: Double = 0.0
) : Serializable
